tests/by-util/test_mv.rs

```diff
@@ -1556,6 +1556,14 @@ fn test_mv_dir_into_file_where_both_are_files() {
         .stderr_contains("mv: cannot stat 'a/': Not a directory");
 }
 
+#[test]
+fn test_mv_dir_into_path_slash() {
+    let scene = TestScenario::new(util_name!());
+    let at = &scene.fixtures;
+    at.mkdir("a");
+    scene.ucmd().arg("a").arg("e/").succeeds();
```

Comment:
This only checks whether the command succeeds but not if the result is correct, that is, `a` is now a subdirectory of `e`.